modules/ROOT/pages/fd-publish-app-as-template.adoc

```diff
@@ -0,0 +1,43 @@
+= Share a Mule Application as a Template on Exchange
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can publish a Mule application as a template on Exchange, which makes it easy for you and other people to create similar Mule applications in the future.
+
+== Benefits of sharing a Mule application as a template
+
+Publishing Mule applications as templates is a great way to share your work with other people. When they need to build a similar application, they can get a head start by downloading and using your template to start their own project. If you accompany your template in Exchange with a good description, screenshots, and perhaps even a video, those other people could get going even more quickly.
```

Comment:
Repetitive start word...they can _begin_ by downloading and using your template to start their own project.

Reply:
"Get a head start" is idiomatic English and has a meaning that is different from "begin". However, I shouldn't use idiomatic English. I'll change this.